docs/modules/maintain-cluster/pages/restart-cluster.adoc

```diff
@@ -0,0 +1,219 @@
+= Restart a cluster
+:description: Static configuration is configuration that cannot be changed while members are running. To apply updates to static configuration, you must restart the cluster.
+
+{description}
+
+There are two options for restarting a cluster:
+
+* <<rolling>>
+* <<whole>>
+
+Where possible, you should perform a rolling restart. In a rolling restart, members are stopped one at a time and each member's data is automatically repartitioned across the remaining cluster members before shutting down. This minimizes the risk of service interruptions and data loss. When the replacement member joins the cluster, data is partitioned across the cluster again.
+
+Depending on the scope of your configuration changes, you may need update a single member, a cluster, or multiple clusters. The process is the same in all cases.
+
+TIP: You can also use the rolling restart process for canarying: testing changes on a single member before updating the rest of the cluster.
+
+Some configuration needs to be consistent across all members in a cluster. To update this configuration, you must restart the whole cluster simultaneously. This causes all data in the cluster to be lost. To avoid data loss, you must enable xref:storage:persistence.adoc[persistence]. To avoid downtime, you must divert traffic to another Hazelcast cluster before restarting.
+
+Note that members are not restarted. In all cases, members are destroyed, and new members are created to replace them.
+
+You should perform restarts in a low utilization or maintenance window to reduce the risk of service impacts.
+
+== Persistence
+
+If xref:storage:persistence.adoc[persistence] is enabled, replacement members will adopt the UUIDs of the destroyed members and attempt to load persisted data from disk. This avoids the repartitioning process, helping the cluster recover more quickly during a rolling restart.
```

Comment:
> This avoids the repartitioning process

only if you switch the cluster to `NO_MIGRATION` or `FROZEN` state (eg. https://docs.hazelcast.com/hazelcast/5.5/kubernetes/kubernetes-persistence#configuration)